backend/frontend/templates/frontend/auth/page_forgot_password.html

```diff
@@ -0,0 +1,15 @@
+{% extends "frontend/auth/base.html" %} {% block content %}
+<div class="mx-auto w-1/3 p-6 bg-white rounded">
+  <h1 class="{{ styles.heading1 }}">Forgot Password</h1>
+  <hr class="my-2" />
+  {% for field, error in form.non_field_errors.items %}
```

Comment:
Will this work? Do non-field errors have fields? 

You do this differently in the different templates, eg: 
```
{% if form.non_field_errors %}
    <div class="border border-red-500 bg-red-100 rounded p-3 mb-4 text-sm">
      {% for error in form.non_field_errors %}
          <p class="text-red-500">{{ error }}</p>
      {% endfor %}
    </div>
  {% endif %}
```

I wonder if there is a way to put the repetitive form error stuff in another template and just include that as needed.

Reply:
Scratched this because {{ form.as_p }} will render the erorrs in a list with custom classes, maybe we can just write styles for those